src/java.base/share/classes/sun/security/x509/X509CertImpl.java

```diff
@@ -468,35 +466,31 @@ public static X509CertImpl newSigned(X509CertInfo info, PrivateKey key, String a
     public static X509CertImpl newSigned(X509CertInfo info, PrivateKey key, String algorithm, String provider)
             throws CertificateException, NoSuchAlgorithmException,
```

Comment:
The javadoc above states that CertificateException is thrown on encoding errors. Will there still be encoding errors? The updated code seems not.

Reply:
Ok.